pages/data-providers/[data-provider-id]/doi.jsx

```diff
@@ -6,14 +6,33 @@ import NumericValue from 'components/numeric-value'
 import ExportButton from 'components/export-button'
 import EnrichmentChart from 'components/enrichment-chart'
 import texts from 'texts/doi'
-import { Card } from 'design'
+import { Card, Icon } from 'design'
 
 // TODO: Remove once cards are in @oacore/design
 // eslint-disable-next-line
 import styles from './doi.css'
 
 const formatPercent = (number, precision = 2) => `${number.toFixed(precision)}%`
 
+const formatDOI = entity => {
```

Comment:
Not sure if you know but you there is a better way of doing this. By extending Table.Column. But basically it's the same. We maybe just want to get rid of `getter` in the future so this approach would be better. It's up to you I don't have preference here.

```jsx
class DOIColumn extends Table.Column {
	render() {
      const { context: {crossrefDoi, repoDoi: originDoi} } = this.props
	  if (crossrefDoi && !originDoi) {
    	return (
	      <>
	        {crossrefDoi}{' '}
	        <Icon
	          className={styles.newIcon}
	          src="/design/icons.svg#new"
	          alt="New"
	        />
	      </>
	    )
	  }
	  return originDoi
	}
}
```

Reply:
I would leave this on you in the future refactoring if you do not insist. 

P.S. React guys would not like extending, do they? 🙂

Reply:
I don't really know but for example, the whole material-ui library is built on extended components if I remember correctly. To be honest I am more functional guy and don't like classes in JS.